nnpackage/schema/circle_schema.fbs

```diff
@@ -1064,6 +1064,9 @@ table Operator {
   // computation, which may require relevant metadata such as quantization
   // parameters.
   intermediates:[int];
+
+  // For profiling
+  p_index:int;
```

Comment:
@mhs4670go , can you explain about this?

Reply:
@seanshpark
This draft is related tunnel T/F. For profiling, each operator in circle model has its id. That is why `p_index`(profiling_index) is added in schema. But since there is no exact decision yet about spec, I made this PR for experiment.

Reply:
I know the background. I am asking why you added this item? from what requirement that ended up adding this.

Reply:
I come up with these two things to solve this requirement(https://github.com/Samsung/ONE/issues/5794#issuecomment-766536492). And this draft is one of them. It could be different implementation.